pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -62,6 +67,30 @@ func NewNodePerformance(profile *performancev2.PerformanceProfile) (*tunedv1.Tun
 		}
 	}
 
+	if profile.Spec.HardwareTuning != nil {
+		isolatedCpuSet, err := cpuset.Parse(string(*profile.Spec.CPU.Isolated))
+		if err != nil {
+			return nil, err
+		}
+		isolatedCpuString := components.ListToString(isolatedCpuSet.List())
+		// converts a string to a string slice
+		isolatedCpuList := strings.SplitN(isolatedCpuString, ",", len(isolatedCpuString))
```

Comment:
Hi @yanirq, based on your comment on unit test for `isolatedCpuList`, the final `isolatedCpuList` will always be an array. Do you want me to create a new generic function which returns a cpulist based on the input provided, and add unit tests for that? Also the `range` go template function can only range over an array but not over lists separated by comma.

Reply:
In that case I think we are good